src/legacy.py

```diff
@@ -0,0 +1,120 @@
+    vasp_xml: str,
+    *,
+    diff_elem: str,
+    material: str,
+    supercell: tuple[int, int, int] | None = None,
+    equil_time: float = 2.5E-12,
+    diffusion_dimensions: int = 3,
+    z_ion: int = 1,
+    nr_parts: int = 10,
+    dist_collective: float = 4.5,
+    density_resolution: float = 0.2,
+    jump_res: float = 0.1,
+    rdf_res: float = 0.1,
+    rdf_max_dist: int = 10,
+    start_end: tuple[int, int] = (5000, 7500),
+    nr_steps_frame: int = 5,
+) -> tuple[SimulationData, SitesData, SimpleNamespace]:
+    """Analyse md data.
+
+    Parameters
+    ----------
+    vasp_xml : str
+        Simulation data input file
+    diff_elem : str
+        Name of diffusing element
+    material : str
+        Name of known material with sites for diffusing elements
+    supercell : tuple[int, int, int], optional
+        Super cell to apply to the known material
+    equil_time : float, optional
+        Equilibration time in seconds (1e-12 = 1 picosecond)
+    diffusion_dimensions : int, optional
+        Number of diffusion dimensions
+    z_ion : int, optional
+        Ionic charge of diffusing ion
+    nr_parts : int, optional
+        In how many parts to divide your simulation for statistics
+        dist_collective : float, optional
+                Maximum distance for collective motions in Angstrom
+    density_resolution : float, optional
+        Resolution for the diffusing atom density plot in Angstrom
+    jump_res: float
+        Resolution for the number of jumps vs distance plot in Angstrom
+    rdf_res : float, optional
+        Resolution of the rdf bins in Angstrom
+    rdf_max_dist : int, optional
+        Maximal distanbe of the rdf in Angstrom
+    start_end : tuple[int, int], optional
+        The time steps for which to start and end the movie
+    nr_steps_frame : int, optional
+        How many time steps per frame in the movie, increase to get shorter and faster movies
+
+    Returns
+    -------
+    tuple[SimulationData, SitesData, SimpleNamespace]
+        Output data
+    """
+    data = SimulationData.from_vasprun(vasp_xml)
+
+    extras = data.calculate_all(
+        equilibration_steps=1250,
+        diffusing_element=diff_elem,
+        z_ion=z_ion,
+        diffusion_dimensions=diffusion_dimensions,
+        n_parts=nr_parts,
+    )
+
+    sites_structure = load_known_material(material, supercell=supercell)
+
+    sites = SitesData(sites_structure)
+    sites.calculate_all(data=data,
+                        extras=extras,
+                        dist_collective=dist_collective)
+
+    plot_all(data=data,
+             sites=sites,
+             **vars(extras),
+             show=False,
+             jump_res=jump_res)
+
+    rdfs = calculate_rdfs(
+        data=data,
+        sites=sites,
+        diff_coords=extras.diff_coords,
+        n_steps=extras.n_steps,
+        equilibration_steps=extras.equilibration_steps,
+        max_dist=rdf_max_dist,
+        resolution=rdf_res,
+    )
```

Comment:
guard with if statement:
```suggestion
    if enable_rdf:
        rdfs = calculate_rdfs(
            data=data,
            sites=sites,
            diff_coords=extras.diff_coords,
            n_steps=extras.n_steps,
            equilibration_steps=extras.equilibration_steps,
            max_dist=rdf_max_dist,
            resolution=rdf_res,
        )
```